samples/backups.js

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Imports the Google Cloud client library
+const {Bigtable} = require('@google-cloud/bigtable');
+const uuid = require('uuid');
+const {inspect} = require('util');
+
+function generateBackupId() {
+  return `gcloud-tests-${uuid.v4()}`.substr(0, 48);
+}
+
+async function runBackupOperations(
+  instanceID,
+  clusterID,
+  tableID,
+  optionalBackupID
+) {
+  const bigtable = new Bigtable();
+  const instance = bigtable.instance(instanceID);
+  const cluster = instance.cluster(clusterID);
+
+  const backupID = optionalBackupID || generateBackupId();
+
+  // [START bigtable_create_backup]
+  const table = instance.table(tableID);
+  const expireTime = new Date(Date.now() + 7 * 60 * 60 * 1000); // 7 hours from now
+
+  // expireTime can be either a Date, PreciseDate, or an ITimestamp
+  const [backupOperation] = await cluster.createBackup(table, backupID, {
+    expireTime,
+  });
+  console.log(
+    'Started a table backup operation:',
+    inspect(backupOperation.latestResponse.name, {depth: null, colors: true})
```

Comment:
Typically we don't drop down to `util.inspect`, but just drop individual properties in the logs.  Since this is a `name`, is it just a `string` type?